cura/BuildVolume.py

```diff
@@ -746,7 +746,8 @@ def _updateDisallowedAreas(self):
                                 break
                         if prime_tower_collision: #Already found a collision.
                             break
-                        if ExtruderManager.getInstance().getResolveOrValue("prime_tower_brim_enable"):
+                        if (ExtruderManager.getInstance().getResolveOrValue("prime_tower_brim_enable") and
```

Comment:
```suggestion
                        if ExtruderManager.getInstance().getResolveOrValue("prime_tower_brim_enable") and
```
Nitpicking; The () around a if statement are not needed.

Reply:
Turns out I did those brackets because line-breaks at that position are otherwise not allowed in python without a `\`, which I think is uglier than the brackets. Had to revert these changes, except for the 'raft' => "raft" ones.